### PR TITLE
Add canvas for a proper Python module

### PR DESCRIPTION
**Pull request by frankrousseau**
Hello there!

To initiate this module, I wrote a basic structure for a good Python module. I added some functions  like `get_[facet]` for each facet and `get_product(barcode)` to fetch data from a given product.
Everything is included in this PR.

I didn't open an issue because we already discussed this live last Thursday. Feel free to close this PR if you think it's not relevant anymore.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    name='openfoodfacts',
+    version=version,
+    description='Easy querying of OpenFoodFacts.',
+    long_description=readme,
+    author='Pierre Slamich',
+    author_email='pierre@openfoodfacts.org',
+    url='http://openfoodfacts.org',
+    packages=packages,
+    package_data={'': ['LICENSE']},
+    package_dir={'hungergame': 'hungergame'},
```
Comment (teolemon) [inline setup.py]: can we get rid of the hungergames thingie

File: openfoodfacts/utils.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import requests
+
+API_URL = "http://world.openfoodfacts.org/"
+
```
Comment (teolemon) [inline openfoodfacts/utils.py]: We could move it to a settings file to prepare for Open Beauty Facts

File: tests/facets_test.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import openfoodfacts
+import requests
+import requests_mock
+
+
+class TestFacets(unittest.TestCase):
+
+    def test_get_traces(self):
+	with requests_mock.mock() as mock:
+	    mock.get('http://world.openfoodfacts.org/traces.json',
```
Comment (teolemon) [inline tests/facets_test.py]: moving the url to settings to avoid rewriting the tests for OBF

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+with open('README.md', 'r') as f:
+    readme = f.read()
+
+setup(
+    name='openfoodfacts',
+    version=version,
+    description='Easy querying of OpenFoodFacts.',
+    long_description=readme,
+    author='Pierre Slamich',
```
Comment (teolemon) [inline setup.py]: Are multiple authors possible ?

File: README.md [hunk 5]
```diff
@@ -1 +1,22 @@
-# openfoodfacts-python
\ No newline at end of file
+# Open Food Facts client for your python scripts
+
+
+### install
+
+
+    sudo pip install git+https://github.com/frankrousseau/openfoodfacts
```
Comment (teolemon) [inline README.md]: If we merge, we need to change it to openfoodfacts-python

File: setup.py (hunk 4 above)
Comment (frankrousseau) [inline setup.py]: Yes, you can add authors by separating them with `, `.

File: openfoodfacts/utils.py (hunk 2 above)
Comment (frankrousseau) [inline openfoodfacts/utils.py]: In a lib it's not recommende to use settings file. What I propose instead is to allow to change this url via a function.

File: setup.py (hunk 1 above)
Comment (frankrousseau) [inline setup.py]: I will fix this in a new PR.

File: README.md (hunk 5 above)
Comment (frankrousseau) [inline README.md]: I will fix this in a new PR.

**Pull request merged**